CMake/FindMKL.cmake

```diff
@@ -87,7 +87,8 @@ try_compile(HAVE_MKL_FFTW3 ${CMAKE_BINARY_DIR}
 
 IF ( HAVE_MKL )
   SET( MKL_FOUND 1 )
-  SET( MKL_FLAGS ${MKL_FLAGS} )
+  SET( MKL_FLAGS ${MKL_COMPILE_DEFINITIONS} )
+  include_directories( ${MKL_INCLUDE_DIRECTORIES} )
   IF ( HAVE_MKL_FFTW3 )
     FILE(WRITE "${CMAKE_CURRENT_BINARY_DIR}/include/fftw3.h" "#include <fftw/fftw3.h>\n" )
```

Comment:
Can we get a chance to avoid the file creation?

Reply:
Sorry I've been busy.  Yes I think we can avoid it, I was just going for minimal change.

Reply:
Good. Please remove those lines and check if Intel +MKl and non-Intel +MKL is still fine. Then we will settle the manual. I hope we don't need to touch this part for a long time.